.circleci/config.base.yml

```diff
@@ -830,12 +830,6 @@ jobs:
             - amplify-cli-ssh-deps-{{ .Branch }}
       - restore_cache:
           key: amplify-pkg-binaries-{{ .Branch }}-{{ .Revision }}
-      - run:
-          name: Upload Pkg Binary
-          command: |
-            source .circleci/local_publish_helpers.sh
-            uploadPkgCli
-            ./out/amplify-pkg-linux-x64 --version
```

Comment:
see https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/20407/workflows/083e1f45-095b-47e5-8465-241f1968bf35/jobs/965535
and https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/20407/workflows/083e1f45-095b-47e5-8465-241f1968bf35/jobs/965538

it gets uploaded 2 times in same release pipe